src/magic_folder/test/cli/common.py

```diff
@@ -63,7 +63,6 @@ def cli(config_directory, argv):
     try:
         try:
             options.parseOptions([
-                b"--debug",
```

Comment:
Does this mean we're going to get less debug output when a cli test fails?  Is this change necessary?

Reply:
Well, so the output you do get with `--debug` turned on isn't very useful -- that's why I changed `cli.run_magic_folder_options` to let the exception "escape". Without *that* change you just always get a traceback to the `maybeDeferred` call .. not to the actual problem.

So, but then instead the CLI runner will get "real" exceptions out instead of printing what the user will see. So I figured since this is all in the "cli testing" helper code the tests should really be expecting to see what the user will. Thinking now, it's certainly possible to have this helper catch the exception and dump the stack "just like python might have done" ... but also none of our tests currently look for printed-out-exceptions from `--debug`.